pyodide-build/pyodide_build/buildpkg.py

```diff
@@ -308,29 +307,29 @@ def patch(pkg_root: Path, srcpath: Path, src_metadata: Dict[str, Any]):
         fd.write(b"\n")
 
 
+def unpack_wheel(path):
+    with chdir(path.parent):
+        subprocess.run([sys.executable, "-m", "wheel", "unpack", path.name], check=True)
```

Comment:
We could use it directly without subprocess (with import of wheel inside this function) https://github.com/pypa/wheel/blob/c716c87ffa528ac9a31de3ed3217058848dc2cd2/src/wheel/cli/unpack.py however it's unclear how stable their public API is. I would guess it shouldn't be too high for these two functions https://github.com/pypa/wheel/issues/262